voicevox_engine/model.py

```diff
@@ -261,3 +261,53 @@ class SupportedDevicesInfo(BaseModel):
     cpu: bool = Field(title="CPUに対応しているか")
     cuda: bool = Field(title="CUDA(Nvidia GPU)に対応しているか")
     dml: bool = Field(title="DirectML(Nvidia GPU/Radeon GPU等)に対応しているか")
+
+
+class SupportedFeaturesInfo(BaseModel):
+    """
+    エンジンの機能の情報
+    """
+
+    support_adjusting_mora: bool = Field(title="モーラが調整可能かどうか")
+    support_adjusting_speed_scale: bool = Field(title="話速が調整可能かどうか")
+    support_adjusting_pitch_scale: bool = Field(title="音高が調整可能かどうか")
+    support_adjusting_intonation_scale: bool = Field(title="抑揚が調整可能かどうか")
+    support_adjusting_volume_scale: bool = Field(title="音量が調整可能かどうか")
+    support_adjusting_silence_scale: bool = Field(title="前後の無音時間が調節可能かどうか")
+    support_interrogative_upspeak: bool = Field(title="疑似疑問文に対応しているかどうか")
+    support_switching_device: bool = Field(title="CPU/GPUの切り替えが可能かどうか")
+
+
+class UpdateInfo(BaseModel):
+    """
+    エンジンのアップデート情報
+    """
+
+    version: str = Field(title="エンジンのバージョン名")
+    descriptions: List[str] = Field(title="アップデートの詳細についての説明")
+    contributors: List[str] = Field(title="貢献者名")
+
+
+class LicenseInfo(BaseModel):
+    """
+    依存ライブラリのライセンス情報
+    """
+
+    name: str = Field(title="依存ライブラリ名")
+    version: Optional[str] = Field(title="依存ライブラリのバージョン")
+    license: Optional[str] = Field(title="依存ライブラリのライセンス名")
+    text: str = Field(title="依存ライブラリのライセンス本文")
+
+
+class EngineManifest(BaseModel):
+    """
+    エンジン自体に関する情報
+    """
+
+    manifest_version: int = Field(title="マニフェストのバージョン")
+    name: str = Field(title="エンジン名")
+    icon: str = Field(title="エンジンのアイコンをBASE64エンコードしたもの")
+    default_sampling_rate: int = Field(title="デフォルトのサンプリング周波数")
+    term_of_service: str = Field(title="エンジンの利用規約")
```

Comment:
```suggestion
    terms_of_service: str = Field(title="エンジンの利用規約")
```

用語としては複数形の`Terms of Service`な気がしました！

- <https://www.google.com/search?q=%22Term+of+Service%22>（約 142,000,000 件）
- <https://www.google.com/search?q=%22Terms+of+Service%22>（約 8,410,000,000 件）